src/ui/analyse/AnalyseCtrl.ts

```diff
@@ -134,6 +135,12 @@ export default class AnalyseCtrl {
         return false
       }
 
+      if (this.data.game.variant.key !== 'antichess' &&
+          this.data.game.variant.key !== 'horde' &&
+          !positionLooksLegit(this.data.game.fen)) {
+        return false
+      }
```

Comment:
How is it done on the website?

Reply:
The website just allows computer evaluation for illegal positions, but when it crashes, it just emits a JavaScript WASM console error instead of crashing the whole site

Reply:
ah ok, thanks for the reply

Reply:
I ended up following your suggestion: https://github.com/veloce/lichobile/commit/a711abb4b643e28183b3c8f0397ed033a143ccdd